src/Language/Mulang/Inspector/Functional.hs

```diff
@@ -56,13 +57,9 @@ usesComprehension = usesForComprehension
 
 -- | Inspection that tells whether an expression uses a lambda expression
 -- in its definition
-usesLambda :: Inspection
-usesLambda = unmatching usesLambdaMatching
-
-usesLambdaMatching :: Matcher -> Inspection
-usesLambdaMatching matcher = containsExpression f
-  where f (Lambda _ e) = matcher [e]
-        f _            = False
+(usesLambda, usesLambdaMatching, countLambdas) = deriveUses f :: InspectionFamily
```

Comment:
what.

How am I finding out you can do this just now?

Haskell never ceases to amaze.

Reply:
Neither did i know! I just assumed it would and it did xD